bin/generateSingleArmResponses.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+uuid1=$(uuidgen)
```

Comment:
Could we add a README detailing how to use this file in the context of testing  DMP-3292. It would be useful for newbies like me

Reply:
Added echo statement to start of script